### ISSUE TITLE
Add verbose logging option to generate_appcast

### ISSUE THREAD

**Issue by latenitefilms** [sparkle-project/Sparkle#2652]
I'm currently trying to get Sparkle to play nicely in a GitHub Actions workflow.

I just ran into a case where `generate_appcast` seemed to stall for an hour before I aborted.

I'm ASSUMING it's because there was a popup (or similar) related to Keychain Access.

I can hopefully workaround this by `--ed-key-file` instead, as per the documentation, however, it would be super handy if `generate_appcast` could be made to output verbose logging for CI purposes.

**Comment by latenitefilms** [comment]
Turns out this already exists!

https://github.com/sparkle-project/Sparkle/blob/d04faabe7da0d93baea993642197341d681bb8de/generate_appcast/main.swift#L166

**Comment by zorgiepoo** [comment]
Yeah but it's a private option and may not be all that encompassing / formalized. Formalizing it and making it apply for all of generate_appcast's logic might be a reasonable request.
